File: Java Code/Logic_Building_Code/Sep/16Sep/Program16.c
#include<stdio.h>
void main(void){

	long num=9;
	int n;
	printf("Enter a num: ");
	scanf("%d",&n);
	for(int i=1;i<=n;i++){
	
		printf("%ld ",num);
		num=num*10+9;
	}
	printf("\n");
}

File: Java Code/Logic_Building_Code/Sep/16Sep/Program16.java
import java.io.*;
class Program16{

	public static void main(String args[]) throws IOException{
	
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		System.out.println("Enter Number: ");
		long num=9;
		int iNum=Integer.parseInt(br.readLine());
		for(int i=1;i<=iNum;i++){
		
			System.out.print(num+" ");
			num=num*10+9;
		}
		System.out.println();

	}
}
